easybuild/easyconfigs/g/gnuplot/gnuplot-5.4.8-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'ConfigureMake'
+
+name = 'gnuplot'
+version = '5.4.8'
+
+homepage = 'http://gnuplot.sourceforge.net'
+description = """Portable interactive, function plotting utility"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = [('https://sourceforge.net/projects/gnuplot/files/gnuplot/%(version)s', 'download')]
+sources = [SOURCE_TAR_GZ]
+checksums = ['931279c7caad1aff7d46cb4766f1ff41c26d9be9daf0bcf0c79deeee3d91f5cf']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('pkgconf', '1.9.5'),
+    ('Autotools', '20220317'),
+]
+
+dependencies = [
+    ('ncurses', '6.4'),
+    ('cairo', '1.17.8'),
+    ('libjpeg-turbo', '2.1.5.1'),
+    ('libpng', '1.6.39'),
+    ('libgd', '2.3.3'),
+    ('Pango', '1.50.14'),
+    ('libcerf', '2.3'),
+    ('X11', '20230603'),
+    ('Qt5', '5.15.10'),
+    ('Lua', '5.4.6'),
+    ('wxWidgets', '3.2.2.1')
```

Comment:
It really needs Qt *and* wxWidgets??

Reply:
Oh it was introduced here:
https://github.com/easybuilders/easybuild-easyconfigs/pull/17483#issuecomment-1463816755

I disagree with the conclusion though, i think it means we should add `--without-wx` (which i think is the right flag)

(also, why was gnuplot-5.4.6 moved to GCC level randomly?)

Reply:
@Micket I removed the wxWidgets dependency and changed the toolchain to GCC for consistency.

Reply:
I think moving to GCC was a mistake. I actually want to see it moved back to GCCcore
https://github.com/easybuilders/easybuild-easyconfigs/pull/18648

Reply:
You actually added both variants. When updating an easyconfig with the command line, you can use `:gnuplot-5.4.8-GCC-12.3.0.eb` to indicate that you want to remove it.